cfe_internal/enterprise/federation/federation.cf

```diff
@@ -317,8 +326,10 @@ bundle agent clean_when_off
     "user" string => "$(cfengine_enterprise_federation:transport_user.user)";
     "home" string => "$(cfengine_enterprise_federation:transport_user.home)";
 @if minimum_version(3.15)
-    "remote_hubs_table_row_count" string => execresult(`$(sys.bindir)/psql cfsettings --quiet --tuples-only --command "SELECT COUNT(*) FROM remote_hubs" 2>/dev/null`, useshell);
-    "federated_reporting_settings_table_row_count" string => execresult(`$(sys.bindir)/psql cfsettings --quiet --tuples-only --command "SELECT COUNT(*) FROM federated_reporting_settings" 2>/dev/null`, useshell);
+      "remote_hubs_table_row_count" -> { "CFE-951" }
+        string => execresult(`cd /tmp; $(sys.bindir)/psql cfsettings --quiet --tuples-only --command "SELECT COUNT(*) FROM remote_hubs" 2>/dev/null`, useshell);
```

Comment:
why `cd tmp` here since you are not containing it with `cfpostgres` user right? It "just works" as root I think because we have a root role.

https://github.com/cfengine/buildscripts/blob/master/packaging/common/cfengine-hub/postinstall.sh#L775-L777

```
    if ! echo "$db_user_list" | grep root >/dev/null; then
      su cfpostgres -c "$PREFIX/bin/createuser -d -s -w root"
    fi
```

Reply:
ack